### ISSUE TITLE
What means about 'importance-weighted' in paper？

### ISSUE THREAD

**Issue by miss-rain** [GT-RIPL/AlwaysBeDreaming-DFCIL#4]
Nice work, but i have some little question.

**What means about 'importance-weighted' in paper?** 

in code, 
**### def data_weighting()** in _default.py,_ 
i find self.dw_k is [1,1,1,1,........1]

**### class AlwaysBeDreaming(DeepInversionGenBN)**  in _datafree.py_
the loss_kd = self.kd_criterion(logits_KD, logits_KD_past).sum(dim=1) * dw_KD
dw_KD is also [1,1,1,......1]

**it's not useful,and i don't understead where means 'importance-weighted' or 'data weighting'?**

**Comment by miss-rain** [comment]
Thanks ! :) 
Great work!
